### ISSUE TITLE
Post Carousel: Add three settings for Slides to scroll

### ISSUE THREAD

**Issue by Misplon** [siteorigin/so-widgets-bundle#1132]
Add new settings for Slides to Scroll for Mobile, Tablet, and Desktop. We should perhaps add a new Settings section. In that event, Link target and Loop posts should probably be moved into the Settings section.

**Comment by AlexGStapleton** [comment]
What are your thoughts on adding these settings to a Responsive settings group instead of a general Settings group?

Do you have a preference for the setting names and descriptions? How about?

Slides to scroll on Desktop
Slides to scroll on tablet
Slides to scroll on mobile

Description: The number of slides to scroll.

The labels are a bit wordy though. Hm. Maybe the settings group could be called Slides to scroll?

**Comment by Misplon** [comment]
Sure, a Responsive section sounds good.

Let's go with a settings group called "Slides to scroll". Perhaps the below will work.

**Slides to scroll**
Set the number of slides to scroll per navigation click or swipe on mobile devices.

- Desktop
- Tablet
- Mobile
